data/campaigns/World_Conquest/lua/game_mechanics/unittypedata.lua

```diff
@@ -2,11 +2,11 @@ local _ = wesnoth.textdomain 'wesnoth-World_Conquest'
 
 type_infos = {
 	["default"] = {
-		founddialogue = _"You guys look like you could use some help. Mind if I join in? It's been a while since I had a good fight!",
+		founddialogue = _"You guys look like you could use some help. Mind if I join in? It’s been a while since I had a good fight!",
 		reply = _ "Excellent. We could always use more help.",
 	},
 	["Orcish Grunt"] = { 
-		founddialogue=_"'bout time. Been forever since I had a good fight, eh?",
+		founddialogue=_"’bout time. Been forever since I had a good fight, eh?",
```

Comment:
It's not an issue for the current PR to solve, but wmlxgettext doesn't pick up any hints about context from the coding style.

Commenting on this line because, seeing only the string without context, I thought this sounded like a Fencer rather than an Orc.